experiments/landlords/parent_company_search/How to Find Building Owners.ipynb

```diff
@@ -0,0 +1,675 @@
+    "\n",
+    "Steps 1 and 2 are done programmatically. Step 3 is done by hand. The steps are outlined below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Be sure to run `pip install . ` in the root directory to access utils modules\n",
+    "from utils import owners\n",
+    "from utils import geo\n",
+    "from utils import parcel_owners"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import requests\n",
+    "import json\n",
+    "import os\n",
+    "import re\n",
+    "import geopandas as gp\n",
+    "import urllib.parse"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 0: Clean up Seattle OSE Data and Identify Buildings of Interest\n",
+    "Seattle's [Office of Sustainability and Environment](https://www.seattle.gov/environment) has released emissions data for buildings in the city from 2015 to 2021. Building owners over 20,000 sq.ft. are required to self-report annually to the city of Seattle. Features of the data include:\n",
+    "- building name\n",
+    "- address\n",
+    "- tax parcel identification number\n",
+    "- council district the building is located within\n",
+    "- buliding type\n",
+    "- different metrics for emissions. \n",
+    "More metadata about the energy benchmarking datasets can be found [here](https://data.seattle.gov/Community/2021-Building-Energy-Benchmarking/bfsh-nrm6).\n",
+    "\n",
+    "When doing initial exploratory analysis, we noticed that not all of the council district codes were correct. To rectify this we got the official council district boundaries form [Seattle GeoData Portal](https://data-seattlecitygis.opendata.arcgis.com/datasets/seattle-city-council-districts-for-council-members-serving-through-2023/explore) in the form of a geojson file [`Council_Districts.geojson`](../../../data/Council_Districts.geojson). For each address in the OSE building efficiency dataset, we convert it to a (latitude, longitude) point and confirmed with which council distirct it belonged to. To learn more, refer to `utils/owners.py`.\n",
+    "\n",
+    "For the purpose of the report, we focused our search on buildings in the downtown neighborhood. We decided this for political reasons but also because that's where a significant portion of the largest buildings in Seattle are located. The benefit of extensible code is that we can change our minds at any time and select a different subset of the data (perhaps we want to consider a particular council district) and have results within a few minutes. \n"
```

Comment:
We should put this in the very beginning. And add that we only did buildings >100k sq ft.